### ISSUE TITLE
[Bug]: Missing items

### ISSUE THREAD

**Issue by YaBoiSean319** [AllTheMods/ATM-9#1833]
### Possible Fixes

Yes

### Modpack Version

0.2.60

### What happened?

I have noticed several items that are missing on the server that I am hosting, but are present in single player. Mainly allthemodium seeds, vibranium seeds, and unobtanium seeds. As well as magical soil does not have a recipe in jei.

![image](https://github.com/AllTheMods/ATM-9/assets/171507387/d2a873eb-9d7b-4a9b-bfa9-ce948fcb9827)

![image](https://github.com/AllTheMods/ATM-9/assets/171507387/58c23155-d885-4226-a44f-434a88f12514)

when I click on the magical soil, it does not bring up a recipe.

I replaced the kubejs folder several times in the server files, reloaded kubejs, and checked that the seeds are on the enabled list in the .js file

![image](https://github.com/AllTheMods/ATM-9/assets/171507387/ab2b829c-c734-4de4-9065-c5b8ab34a8d8)

Thank you for any help you can provide :)

### latest.log

https://gist.github.com/YaBoiSean319/1aa6fe41de498beb393448dfd1108aeb

**Comment by YaBoiSean319** [comment]
Update:
I installed a fresh server and client atm9. Issue still persists.
I checked the cropInfo.json through my server manager. those crops were not listed under enabled. I added them back, reloaded kubejs. Crops were removed from the list again and still missing in game. I also have been getting a kubejs error message every time I log in. I attached the server.log file from kubejs.

[server.log](https://github.com/user-attachments/files/15923191/server.log)
